packages/parcel-transformer-external/src/__tests__/transformer.parceltest.ts

```diff
@@ -0,0 +1,89 @@
+import { join } from 'path';
+
+import Parcel, { createWorkerFarm } from '@parcel/core';
+import { MemoryFS } from '@parcel/fs';
+import { format } from 'prettier';
+
+const rootPath = join(__dirname, '..', '..', '..', '..');
+
+const workerFarm = createWorkerFarm();
+
+afterAll(() => {
+  workerFarm.end();
+});
+
+const outputFS = new MemoryFS(workerFarm);
+
+const getParcelInstance = (workingDir: string, sourceMap = false) => {
```

Comment:
i don't have enough context on the Parcel stuff to properly review it beyond "seems fine to me", but i'll do what i can to review the parts that are Compiled related